SonarQube.TeamBuild.PreProcessor/ArgumentProcessor.cs

```diff
@@ -30,12 +30,15 @@ private static class KeywordIds
             public const string ProjectKey = "1";
             public const string ProjectName = "2";
             public const string ProjectVersion = "3";
-            public const string RunnerPropertiesPath = "4";
-            public const string DynamicSetting = "5";
+            public const string InstallLoaderTargets = "4";
```

Comment:
I propose to change these IDs from numeric to more meaningful strings, like the IDs in the bootstrapper. Aside from consistency, it would also make debugging easier.